tests/py/dynamo/models/test_models_export.py

```diff
@@ -251,8 +252,9 @@ def calibrate_loop(model):
 
 @unittest.skipIf(
     not importlib.util.find_spec("modelopt")
-    or Version(metadata.version("nvidia-modelopt")) < Version("0.16.1"),
-    "modelopt 0.16.1 or later is required Int8 quantization is supported in modelopt since 0.16.1 or later",
+    or platform.system() != "Linux"
```

Comment:
Probably want to switch this so that the platform check is first and the rest short circuit

Reply:
modified as suggested.